6.CSS/style.css

```diff
@@ -0,0 +1,229 @@
+@font-face {
+    font-family: "Open Sans";
+    src: url(./fonts/OpenSans-Regular.ttf);
+    font-weight: 400;
+}
+
+@font-face {
+    font-family: "Open Sans";
+    src: url(./fonts/OpenSans-Bold.ttf);
+    font-weight: 700;
+}
+
+* {
```

Comment:
all global selectors like this or slection by tag also will be good to store into separated file global.css

Reply:
in general, you should rarely use selection by tag name or by id always try to use class